specification/purview/data-plane/Azure.Analytics.Purview.DataMap/stable/2023-09-01/purviewdatamap.json

```diff
@@ -0,0 +1,7267 @@
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. On successful update of requested type definitions.",
+            "schema": {
+              "$ref": "#/definitions/AtlasTypesDef"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Type"
+        ],
+        "description": "Delete API for all types in bulk.",
+        "operationId": "Type_BulkDelete",
+        "x-ms-examples": {
+          "Type_BulkDelete": {
+            "$ref": "./examples/Type_BulkDelete.json"
+          }
+        },
+        "parameters": [
+          {
+            "in": "body",
+            "name": "typesDef",
+            "description": "A composite object that captures all types to be deleted",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AtlasTypesDef"
+            }
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "OK. On successful deletion of the requested type definitions."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/atlas/v2/types/typedefs/headers": {
+      "get": {
+        "tags": [
+          "Type"
+        ],
+        "description": "List all type definitions returned as a list of minimal information header.",
+        "operationId": "Type_ListHeaders",
+        "x-ms-examples": {
+          "Type_ListHeaders": {
+            "$ref": "./examples/Type_ListHeaders.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/includeTermTemplate"
+          },
+          {
+            "$ref": "#/parameters/type"
+          },
+          {
+            "$ref": "#/parameters/purviewAPIVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "An array of AtlasTypeDefHeader matching the search criteria\nor an empty list if no match.",
+            "schema": {
+              "type": "array",
+              "description": "An array of AtlasTypeDefHeader matching the search criteria\nor an empty list if no match.",
+              "items": {
+                "$ref": "#/definitions/AtlasTypeDefHeader"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/types/termtemplatedef/guid/{guid}": {
```

Comment:
I didn't see a PUT of `/types/termtemplatedef/guid/{guid}`. How's it created?

Reply:
all type defs are written by native Atlas API Post/Put /atlas/v2/types/typedefs